lib/mayaUsd/ufe/UsdUndoDuplicateCommand.cpp

```diff
@@ -88,6 +90,7 @@ bool UsdUndoDuplicateCommand::duplicate(const SdfLayerHandle& layer, const SdfPa
 	// We use the source layer as the destination.  An alternate workflow
 	// would be the edit target layer be the destination:
 	// layer = self._stage.GetEditTarget().GetLayer()
+	MayaUsd::ufe::InAddOrDeleteOperation ad;
```

Comment:
Without this in StagesSubject we'll end up sending a SubtreeInvalidate notif which is not correct. When duplicating we need an ObjectAdd. You'll see below on line 114, we explicitly send and ObjectDelete.

Reply:
I debated between putting this notif here or in the redo() below - which calls this static method. I think its better here because this is also called from the sceneitemops.